lib/new_relic/telemetry_sdk.rb

```diff
@@ -1,8 +1,20 @@
+# encoding: utf-8
+# frozen_string_literal: true
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
 require 'net/http'
 require 'json'
 require 'zlib'
 require 'securerandom'
 
+module NewRelic
+  module TelemetrySdk
+    USER_AGENT_NAME = "NewRelic-Ruby-TelemetrySDK"
+    RFC7230_TOKEN = /^[!#$%&'*+\-.^_`|~0-9A-Za-z]+$/
```

Comment:
Thank you for diving into the details of this token in the specification with the team.